src/containers/app/AppHeaderContainer.js

```diff
@@ -269,6 +278,6 @@ function mapDispatchToProps(dispatch :Function) :Object {
   };
 }
 
-export default withRouter(
+export default withRouter<any>(
```

Comment:
Should probably just make this `<*>` like the others.

Reply:
Missed this one, thanks. By the way, `*` is apparently deprecated.

https://github.com/facebook/flow/issues/6308

Reply:
Missed this one, thanks. By the way, `*` is apparently deprecated.